### ISSUE TITLE
frei0r openCV plugins facebl0r and facedetect cause segmentation fault

### ISSUE THREAD

**Issue by sxe** [mltframework/mlt#315]
Hi, 

not sure this is the right place to post this but it made the most sense to me. If not, please let me know what would be a more appropriate place to report this.

I noticed this because kdenlive and shotcut both weren't able to export videos. 
By deleting frei0r plugins one by one I figured out, that as soon as I delete the facebl0r and facedetect plugins everything works fine.

I am on archlinux and this happens with the latest mlt / kdenlive / shotcut / frei0r versions from the repository AND the latest git versions.

Not sure what other information would be useful. Please let me know if you need anything else.

Cheer

**Comment by ddennedy** [comment]
I do not believe this is a MLT code bug. More likely it is a build problem such as a conflict between MLT and OpenCV dependencies.

**Comment by sxe** [comment]
What would you suggest me doing? In the end it's a problem for everyone starting by the Linux distribution I am using.

**Comment by ddennedy** [comment]
One short term fix is to not build frei0r against OpenCV. The filters it provides are not very popular.
Another thing to do is to use ldd against libmltqt.so and facebl0r.so to see if they are using the same versions of Qt. Or, use similar methodology or gdb to figure out another source of conflict. Find someone in your community who know anything about C or C++ to investigate it for your distro, which I am not running.
